### DISCUSSION TITLE
Newcomer having trouble

### DISCUSSION THREAD

**Discussion by derekoharrow** [netalertx/NetAlertX#657] [General]
Hi,
Just discovered NetAlertX and love it - could be a great replacement for Fing!
I've got it installed and running in Docker on a Ubuntu host on top of ESXi.

I configured the subnet (192.168.1.0/24 on ens160) and it found 59 devices. But, I have two issues:

Firstly, It works for a while, but then all of a sudden, 57 of those 59 devices are showing as Offline.
I have the arp scanner setup to scan on a schedule of `*/3 * * * *`
They seem to come back after a while, but then go offline again. Many of these devices are permanently online, so should be visible always.

Secondly, the UI mostly works, but every so often it seems to screw-up - looks like some kind of caching issue? Either the web page keeps refreshing itself, or elements are empty (drop downs, dialogues, etc).

Any ideas how to start debugging and resolving these problems?

Thanks!

**Comment by jokob-sk** [comment]
Hi there!

Thanks for trying the app!

I think the first issue is related to out of sync schedules. Check this doc for details: https://github.com/jokob-sk/NetAlertX/blob/main/docs/NOTIFICATIONS.md

I'm aware of some caching issues and it's on to out my list to get to once I have time and mental bandwidth for it. Any help is appreciated if someone wants to help me to improve the app quicker. 🙏 

Edit: Usually as a workaround clicking the refresh button in the top right corner helps.

Hope this helps,
J

**Comment by derekoharrow** [comment]
Thanks - aligning the discovery sync schedules seems to have settled things down.
